eZ/Bundle/EzPublishRestBundle/Tests/Functional/RoleTest.php

```diff
@@ -490,7 +490,7 @@ public function testUnassignRoleFromUser($roleAssignmentHref)
      *
      * @return string role assignment href
      */
-    public function testAssignRoleToUserGroup($roleHref)
+    public function _testAssignRoleToUserGroup($roleHref)
```

Comment:
I need to rephrase that a bit. This particular test breaks roles in the database, and prevents further tests from working.